src/main/java/edu/hw3/task1/AtbashCipherUtils.java

```diff
@@ -0,0 +1,32 @@
+package edu.hw3.task1;
+
+public final class AtbashCipherUtils {
+    private AtbashCipherUtils() {
+    }
+
+    public static String atbash(String input) {
+        StringBuilder result = new StringBuilder();
+
+        for (char c : input.toCharArray()) {
+            if (Character.isLetter(c)) {
+                result.append(getMirroredChar(c));
+            } else {
+                result.append(c);
+            }
+        }
+
+        return result.toString();
+    }
+
+    private static char getMirroredChar(char c) {
+        char mirroredChar;
+
+        if (Character.isUpperCase(c)) {
+            mirroredChar = (char) ('Z' - (c - 'A'));
```

Comment:
`Enhacnement` можно сразу делать return 
```suggestion
            return (char) ('Z' - (c - 'A'));
```